i18n/translation-en.json

```diff
@@ -146,7 +146,7 @@
     "passwordRequired": "Password required",
     "preparingToPrint": "Preparing to print...",
     "annotationReplyCount": "{{count}} Reply",
-    "annotationReplyCount_plural" : "{{count}} Replies",
+    "annotationReplyCount_plural": "{{count}} Replies",
```

Comment:
Let's follow https://github.com/XodoDocs/webviewer/wiki/WebViewer-UI-Localization to have the text in other languages.